code/modules/mob/living/simple_animal/guardian/types/fire.dm

```diff
@@ -1,12 +1,12 @@
 //Fire
 /mob/living/simple_animal/hostile/guardian/fire
 	a_intent = "help"
-	melee_damage_lower = 10
-	melee_damage_upper = 10
+	melee_damage_lower = 7
```

Comment:
The only time they kill anyone is on thr shuttle anyway.

Actually I rarely see anything other than chaos or healing these days since security finally figured out laser guns